tests/support/stub/SiteController.php

```
@@ -0,0 +1,202 @@
+        $this->response->format = Response::FORMAT_RAW;
+
+        $tmpFile = tmpfile();
+
+        if ($tmpFile === false) {
+            throw new Exception('Failed to create temporary file');
+        }
+
+        fwrite($tmpFile, 'This is a test file content.');
+        rewind($tmpFile);
+
+        $tmpFilePath = stream_get_meta_data($tmpFile)['uri'];
```

PHPMD (2.15.0)

Avoid unused local variables such as '$tmpFilePath'. (Unused Code Rules)

(UnusedLocalVariable)